lib/strategies/cacheOnly.js

```diff
@@ -14,12 +14,21 @@
 	limitations under the License.
 */
 'use strict';
+var globalOptions = require('../options');
 var helpers = require('../helpers');
 
 function cacheOnly(request, values, options) {
+  options = options || {};
   helpers.debug('Strategy: cache only [' + request.url + ']', options);
   return helpers.openCache(options).then(function(cache) {
-    return cache.match(request);
+    return cache.match(request).then(function(response) {
+      var cacheOptions = options.cache || globalOptions.cache;
+      if (helpers.isResponseFresh(response, cacheOptions.maxAge, Date.now())) {
```

Comment:
I am assuming this is `cacheOptions.maxAge` and **not** `cacheOptions.maxAgeSeconds` to be backward compatible with the current behaviour right?